nengo_loihi/builder.py

```diff
@@ -29,7 +29,7 @@
     CxSpikeInput,
 )
 from nengo_loihi.splitter import ChipReceiveNeurons, ChipReceiveNode
-from nengo_loihi.synapses import CxSynapses
+from nengo_loihi.synapses import Synapses
```

Comment:
So one nice thing about having the `Cx` on the front is that it made it clear that this `Synapses` object was different from `nengo.Synapse`.

If we have that `model` folder I was talking about, then here I would just `import nengo_loihi.model as loihi_model`, so that we can be more explicit with `loihi_model.Synapses`, etc.